### ISSUE TITLE
content-web.yml indent issue

### ISSUE THREAD

**Issue by snpdev** [microsoft/MCW-Cloud-native-applications#217]
The issue with the `content-web.yml` indentation raised by @anushabc in #210 is still a problem.  "Copy/paste from the Markdown" as suggested in https://github.com/microsoft/MCW-Cloud-native-applications/issues/210#issuecomment-751872402 is not a solution - the incorrect indentation in the HTML is a result of incorrect indentation in the markdown.

The following yml works for me:
```
name: content-web

# This workflow is triggered on push to the 'content-web' directory of the  master branch of the repository
on:
  push:
    branches:
      - master
    paths:
      - 'content-web/**'

  # Configure workflow to also support triggering manually
  workflow_dispatch:

# Environment variables are defined so that they can be used throughout the job definitions.
env:
  imageRepository: 'content-web'
  resourceGroupName: 'fabmedical-[SHORT_SUFFIX]'
  containerRegistryName: 'fabmedical[SHORT_SUFFIX]'
  containerRegistry: 'fabmedical[SHORT_SUFFIX].azurecr.io'
  dockerfilePath: './content-web'
  tag: '${{ github.run_id  }}'

# Jobs define the actions that take place when code is pushed to the master branch
jobs:
  build-and-publish-docker-image:
    name: Build and Push Docker Image
    runs-on: ubuntu-latest
    steps:
    # Checkout the repo
    - uses: actions/checkout@master

    - name: Set up Docker Buildx
      uses: docker/setup-buildx-action@v1

    - name: Login to ACR
      uses: docker/login-action@v1
      with:
        registry: ${{ env.containerRegistry }}
        username: ${{ secrets.ACR_USERNAME }}
        password: ${{ secrets.ACR_PASSWORD }}

    - name: Build and push an image to container registry
      uses: docker/build-push-action@v2
      with:
        context: ${{ env.dockerfilePath  }}
        file: "${{ env.dockerfilePath }}/Dockerfile"
        pull: true
        push: true
        tags: |
          ${{ env.containerRegistry }}/${{ env.imageRepository }}:${{ env.tag }}
          ${{ env.containerRegistry }}/${{ env.imageRepository }}:latest
```

**Comment by beifeng2744** [comment]
This issue still exist now.

**Comment by DawnmarieDesJardins** [comment]
Closing issue as resolved with merge of PR #226
